dev/src/MultiGraphTest.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import ContainerDimensions from "react-container-dimensions";
+
+import Graph from "../../src/directed/ForceDirectedGraph";
+
+const graphData = {
```

Comment:
add this to a separate config file maybe

Reply:
sure ... it's only a test file, but I guess it'll make the class cleaner